### DISCUSSION TITLE
Impostos APP

### DISCUSSION THREAD

**Discussion by reinaldo-paiva** [frontendbr/forum#2022] [Perguntas e Respostas]
Pessoal, tudo bem?

Minha duvida é muito simples. quando você desenvolve um APP e coloca a vendas no Google Store ou na Apple Store por US$ 4.00 a unidade, cada vez que é realizado uma venda quanto de imposto e que tipo de imposto você tem de pagar? 
Você paga ICMS? Quanto %
Você paga ISS? Quanto %
Você paga IR? Quanto %
Alguém pode me auxiliar e se possivel com informações técnicas.
 

Agradeço imensamente.
 

Reinaldo Paiva
(19) 98271-3204 - Piracicaba / SP

**Comment by willycamargo** [comment]
Eu **acho** que são as lojas que gerenciam esses impostos, te passando o valor líquido no final. Até porque quando compro um app eu recebo a nota fiscal do google e não da empresa que desenvolveu o app.

Porém, além dos impostos, acho que a cada venda as lojas cobram 30% de taxa de transação em cima valor bruto. Portanto se vender um app por 4,00 no final irá receber 2,80 - impostos.

Aqui os primeiros links que achei no google:

- Google - Visão geral do imposto sobre vendas para comerciantes: https://support.google.com/paymentscenter/answer/7158952
- Google - Taxas de transação para comerciantes: https://support.google.com/paymentscenter/answer/7159343?hl=pt-BR
- Apple - Compra e Ativação: https://developer.apple.com/pt/support/purchase-activation/
- Apple - Apple Store Connect: https://developer.apple.com/pt/support/app-store-connect/

**Comment by reinaldo-paiva** [comment]
Willy, muita gratidão pelo auxilio... ajudou... entretanto, a Receita
federal não gostou! Ela quer a parte dela e ai que está o problema!!!
pois, os trinta por cento (30%) é a parte do Google / Apple.
só que na hora que você vai receber o que eles enviam para você... você tem
de emitir uma nota fiscal... e ai que o bicho pega!!!!
A quem defenda que você não tem que pagar nada!!!
a quem defenda que é um produto e você tem de pagar ICMS
TEm quem acredita que o Imposto é mais 17%... tem quem acredita que é mais
29% e até quem defende que é mais 40%
não existe uma legislação especifica, e você pode até perder o seu dinheiro
por não receber!!!
pois ele vem em dolar e você tem de ir receber através de uma nota fiscal!!!
simples, mas complicado!!!!
principalmente se você falar de ulguns milhares de dolares!!!

Se alguém mais puder auxiliar!!!! agradeço.


Reinaldo Paiva

Em sáb., 20 de fev. de 2021 às 08:34, Willy Camargo <
notifications@github.com> escreveu:
